boa/src/builtins/function/mod.rs

```diff
@@ -37,15 +37,21 @@ pub struct RegularFunction {
     /// The fields associated with the function
     pub object: Object,
     /// This function's expression
-    pub expr: Expr,
+    pub expr: Node,
     /// The argument declarations of the function
-    pub args: Vec<Expr>,
+    pub args: Vec<Node>,
 }
 
 impl RegularFunction {
     /// Make a new regular function
     #[allow(clippy::cast_possible_wrap)]
-    pub fn new(expr: Expr, args: Vec<Expr>) -> Self {
+    pub fn new(expr: Node, f_args: Vec<FormalParameter>) -> Self {
```

Comment:
maybe change the `expr: Node` parameter to `node: Node`.

Reply:
Done!